include/toBeSorted/cc/d_cc_m3d_g_sph.h

```diff
@@ -0,0 +1,23 @@
+#ifndef TO_SORT_D_CC_M3D_G_SPH_H
+#define TO_SORT_D_CC_M3D_G_SPH_H
+
+#include <m/m_vec.h>
+
+class cM3dGSph {
+public:
+    /* 0x00 */ mVec3_c mCenter;
+    /* 0x0C */ f32 mRadius;
+    /* 0x10 */ f32 field_0x10;
```

Comment:
I don't know why a sphere is characterized by another float in additon to the radius.